daml-lf/language/daml-lf.bzl

```diff
@@ -18,17 +18,29 @@ LF_VERSIONS = [
     "1.7",
     "1.8",
     "1.11",
+    "1.12",
     "dev",
 ]
 
 # The subset of LF versions accepted by the compiler in the syntax
 # expected by the --target option.
-COMPILER_LF_VERSIONS = ["1.dev" if ver == "dev" else ver for ver in LF_VERSIONS]
+COMPILER_LF_VERSIONS = [
```

Comment:
Please filter `LF_VERSIONS` instead of making this an explicit list. I deliberately changed this since the two got out of sync.

Reply:
And add a TODO for removing the filter once the compiler side has caught up.